--- a/rc/fzf.kak
+++ b/rc/fzf.kak
@@ -208,5 +208,6 @@
             printf "%s\n" "SHELL=${shell_path}"
         fi
+        printf "%s\n" "export PATH='$(printf "%s" "$PATH" | sed "s/'/'\\\\''/g")'"
         # compose entire fzf command with all args into single file which will be executed later
         printf "%s\n" "export FZF_DEFAULT_OPTS=\"${kak_opt_fzf_default_opts:-}\""
         printf "%s\n" "cd \"${PWD}\" && ${preview_position} ${items_cmd} ${fzf_impl} ${default_query} ${fzf_args} ${preview_cmd} ${filter} > ${result}"
